test/cpp/end2end/xds/xds_utils.cc

```diff
@@ -78,9 +78,12 @@ std::string XdsBootstrapBuilder::Build() {
 }
 
 std::string XdsBootstrapBuilder::MakeXdsServersText(
-    absl::string_view server_uri) {
+    absl::Span<const std::string> server_uris) {
+  std::vector<std::string> server_features;
```

Comment:
Why were these 4 lines moved up to the top of this function?  Seems like it flowed a bit better where they were before.